src/PrestaShopBundle/Form/Admin/Improve/Payment/Preferences/PaymentModuleRestrictionsParentType.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Improve\Payment\Preferences;
+
+use PrestaShop\PrestaShop\Core\Module\Legacy\ModuleInterface;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class PaymentModuleRestrictionsParentType defines is parent for all restriction forms in Payment Preferences
```

Comment:
just a not for future PR's (not blocking ofc)
Comments  like `Class PaymentModuleRestrictionsParentType`... should be avoided. We can already see the class name so such comment just introduces an opportunity to get outdated comment if class is renamed (and doesn't actually provide any info :smile:)